wiki/Accuracy/id.md

```diff
@@ -0,0 +1,76 @@
+# Akurasi
+
+Akurasi adalah suatu pengukur konsistensi pemain. Terdapat tiga aipe akurasi yang dipunyai pemain. Salah satunya yaitu akurasi beatmap yang bergantung pada skor hit yang diperoleh. Lainnya adalah akurasi rata-rata pemain yang ditimbang agar skor yang lebih baik dapat lebih menonjol. Dan terakhir, akurasi [pp](/wiki/pp) pemain yang bergantung pada akurasi skor yang terkirim.
+
+## Mode permainan
+
+### ![](/wiki/shared/mode/osu.png) osu!standard
+
+![Akurasi = (50 \* jumlah 50 + 100 \* jumlah 100 + 300 \* jumlah 300) / 300(jumlah 0 + jumlah 50 + jumlah 100 + jumlah 300)](img/accuracy_standard.png "Formula akurasi untuk osu!standard")
+
+Di osu!standard, akurasi dikalkulasi dengan menimbang penilaian yang diperoleh dari setiap hit obyek berdasarkan nilainya dan dibagi dengan jumlah maksimum yang mungkin.
+
+Referensi untuk satu hit lingkaran:
+
+```
+300 -> 300 / 300 = 1   = 100.00%
+100 -> 100 / 300 = 1/3 =  33.00%
+50  ->  50 / 300 = 1/6 =  16.67%
+0   ->   0 / 300 = 0   =   0.00%
+```
+
+### ![](/wiki/shared/mode/taiko.png) osu!taiko
+
+![Akurasi = 0.5(jumlah GOOD + jumlah GREAT) / (jumlah BAD + jumlah GOOD + jumlah GREAT)](img/accuracy_taiko.png "Formula akurasi untuk osu!taiko")
+
+Di osu!taiko, akurasi dikalkulasikan dengan mengambil jumlah akurasi not dibagi dengan jumlah not. Akurasi not adalah sebagai berikut: sebuah GREAT (良) dihitung sebagai 100%, GOOD (可) sebagai 50% (sebagian), dan MISS/BAD (不可) sebagai 0% (yang juga memutus combo). Drum roll dan pemutar tidak mempengaruhi akurasi.
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+
+![Akurasi = (jumlah tetesan kecil + jumlah tetesan + jumlah buah) / (jumlah tetesan kecil terlewat + jumlah tetesan terlewat + jumlah buah terlewat + jumlah tetesan kecil + jumlah tetesan + jumlah buah)](img/accuracy_catch.png "Formula akurasi untuk osu!catch")
+
+Di osu!catch, akurasi dikalkulasi dengan mengambil jumlah hit obyek tanpa spinner terambil dibagi dengan jumlah hit obyek tanpa spinner. Semua hit obyek mempunyai nilai sama, kecuali pisang, karena mereka merupakan bagian dari putaran.

```

Comment:
` ... karena mereka merupakan bagian dari putaran ... ` 

use `spinner` instead of `putaran`

Reply:
ye